uml/class.puml

```diff
@@ -1,44 +1,73 @@
 @startuml
+
+
 'https://plantuml.com/class-diagram
 
 package commands {
-    interface SelectCommand<T> {
-        +select(columns: String...): T
+
+    interface WhereCommand<T> {
+       +where(condition: String): T
     }
 
-    interface FromCommand<T> {
-        +from(table: String): T
+    interface LimitCommand<T>{
+        +limit(limit: Integer): T
     }
 
-    interface WhereCommand<T> {
-       +where(condition: String): T
+    interface OrderByCommand<T> {
+        +orderBy(columns: String): T
+        +orderBy((columns: String, order: Order): T
     }
+    OrderByCommand --> Order : <<has>>
```

Comment:
Replace with `--+`

Reply:
May need to specify what this relation means?

Reply:
If you think it necessary, yes